pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md

```diff
@@ -18,11 +18,11 @@ From the **Add Cluster** menu, select the **Create Konvoy Cluster** and provide
 - **Workspace**: The workspace where this cluster belongs.
 - **Kubernetes Version**: The initial version of Kubernetes to install on the cluster.
 - **Name**: A valid Kubernetes name for the cluster.
-- **Add Labels**: By default, your cluster has labels that reflect the infrastructure provider provisioning. For example, your AWS cluster may be labelled with the datacenter region and `provider: aws`. Cluster labels are matched to the selectors created for [projects][projects]. Changing a cluster label may add or remove the cluster from projects.
+- **Add Labels**: By default, your cluster has labels that reflect the infrastructure provider provisioning. For example, your AWS cluster may have a label for the data center region and `provider: aws`. Cluster labels are matched to the selectors created for [projects][projects]. Changing a cluster label may add or remove the cluster from projects.
```

Comment:
We'll need documentation on how on the Kommander configmap non-standard regions such as us-gov-east-1, us-iso-east-1.c2s.ic.gov. As far as the labels, please provide examples that the end-user will need to use or modify?

Reply:
I don't believe any labels are required for the user to add, simply optional.

Good call about configuring the custom regions